.github/workflows/release.yml

```diff
@@ -0,0 +1,106 @@
+---
+name: "Release"
+on: # yamllint disable-line rule:truthy rule:comments
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    name: "Build package with poetry"
+    runs-on: "ubuntu-24.04"
+    if: "startsWith(github.ref, 'refs/tags/v')"
+    steps:
+      - uses: "actions/checkout@v4"
+      - name: "Install poetry"
+        # TODO: Pass poetry version as repo var?
+        run: "pipx install poetry==1.8.5"
+      - name: "Run Poetry Build"
+        run: "poetry build"
+
+      # Is this necessary? Maybe it should be a test instead?
```

Comment:
Instead of doing this (removing `refs/tags/` from `GITHUB_REF`) the next step can just use `run: "poetry version ${{ github.ref_name }}"`

Reply:
I'm wondering why we need this at all? Shouldn't we trust that `pyproject.toml` has the correct version?

Reply:
This is why I was wondering if we could write a one-liner that compares the tag version to the pyproject version and fails if they are different...

Reply:
That works too

Reply:
Maybe something like
```sh
if ["${{ github.ref_name }}" -neq "v${poetry version}"]; then
    exit 1;
fi
```

Reply:
I have been meaning to write this and put into cookie-cutter for a while now. Def agree with this approach.